.github/ISSUE_TEMPLATE/project-onboarding.md

```diff
@@ -31,21 +31,69 @@ Please track your progress by using "Quote reply" to create your own copy of thi
 - [ ] Accept the invite to join the CNCF GitHub Enterprise account. We'll then add `thelinuxfoundation` as an organization owner to ensure neutral hosting of your project.
 - [ ] Migrate your Slack channels (if any) to the [Kubernetes or CNCF Slack workspace](https://slack.com/help/articles/217872578-Import-data-from-one-Slack-workspace-to-another). CNCF staff can help.
 - [ ] Submit a [pull request](https://github.com/cncf/landscape/pulls) to add your project as a Sandbox project to the [Cloud Native Landscape](https://landscape.cncf.io) by updating [landscape.yml](https://github.com/cncf/landscape/blob/master/landscape.yml) following these [instructions](https://github.com/cncf/landscape2/blob/main/docs/config/data.yml).
-- [ ] Transfer [your domain to the CNCF](https://jira.linuxfoundation.org/plugins/servlet/theme/portal/2/create/63). The "LF Stakeholder email" is <projects@cncf.io>.
+- [ ] Transfer [your domain(s) to the CNCF](https://jira.linuxfoundation.org/plugins/servlet/theme/portal/2/create/63). The "LF Stakeholder email" is <projects@cncf.io>.
 - [ ] Transfer any [trademark and logo assets to the Linux Foundation](https://github.com/cncf/foundation/tree/main/agreements).
-- [ ] Submit a [pull request](https://github.com/cncf/artwork) with your artwork.
-- [ ] Transfer website analytics to <projects@cncf.io>. CNCF staff can help.
+- [ ] Submit a [pull request](https://github.com/cncf/artwork) with your existing artwork.  
+_Note_: If you don't have artwork, CNCF can help design some.
+- [ ] Transfer website analytics to <projects@cncf.io>, if applicable. CNCF staff can help.
```

Comment:
If applicable as not all projects will have analytics, and for those that do, it may not be Google Analytics so another transfer method may need to be used.